package.json

```
@@ -155,6 +155,10 @@
                 "type": "string",
                 "description": "A unique identifier for this connection to be used as a reference in `#sonarlint.connectedMode.project#`. Only needed if you plan to use multiple connections to SonarQube (Server, Cloud)."
               },
+              "region": {
+                "type": "string",
```

```suggestion
                "type": "string",
                "enum": ["EU", "US"],
```
This should constrain the possible values and lower the risk of unacceptable ones.